implementations/rust/ockam/ockam_api/src/cloud/enroll.rs

```diff
@@ -148,6 +159,18 @@ pub mod auth0 {
         pub access_token: Token,
     }
 
+    #[derive(serde::Deserialize, Debug)]
+    #[cfg_attr(test, derive(PartialEq, Eq, Clone))]
+    pub struct UserInfo {
+        pub sub: String,
+        pub nickname: String,
+        pub name: String,
+        pub picture: String,
```

Comment:
Is this one expected to be a fully-qualified URL, relative URL, something else?

Reply:
This looks like [a fully qualified URL](https://github.com/build-trust/ockam/issues/5341).